samples/SampleApp/SampleApp.csproj

```diff
@@ -12,6 +12,8 @@
   </ItemGroup>
 
   <ItemGroup>
+    <!-- REVIEW: Having the Https project depend on the non-Core project is breaking since it no longer transitively pulls in UseKestrel() :( -->
```

Comment:
I think that we'll have to leave this as an intentional breaking change. I just wanted to make people aware.